src/js/all/stories/AllComponents.js

```diff
@@ -165,7 +169,7 @@ const Components = () => {
       </FormField>
     </Box>,
     <Box key="time" gap="medium">
-      <Calendar size="small" />
+      <Calendar size="small" level={2} />
```

Comment:
this wont do anything until `level` is added for Calendar?

Reply:
Right now it isn't doing anything but if we merge https://github.com/grommet/grommet/pull/7456 then it will. If we want i could take this out for now and add it in a separate PR?